internal/db/group_test.go

```diff
@@ -143,15 +141,15 @@ func (s *dbSuite) TestUpdateGroup(c *qt.C) {
 	c.Assert(err, qt.IsNil)
 
 	ge1.Name = "renamed-group"
-	err = s.Database.UpdateGroup(context.Background(), ge1)
+	err = s.Database.UpdateGroupName(context.Background(), ge1.UUID, ge1.Name)
 	c.Check(err, qt.IsNil)
 
 	ge2 := &dbmodel.GroupEntry{
 		Name: "renamed-group",
 	}
 	err = s.Database.GetGroup(context.Background(), ge2)
 	c.Check(err, qt.IsNil)
-	c.Assert(ge2, qt.DeepEquals, ge1)
```

Comment:
I think it's because if you use `Model` you correctly update `UpdateAt` each time you change something in the Database Model, so ge2 and ge1 shouldn't be `DeepEquals`